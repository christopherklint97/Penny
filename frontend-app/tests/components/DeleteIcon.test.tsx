import React from 'react'
import { render } from '../testUtils'
import DeleteIcon from '../../components/DeleteIcon'

describe('DeleteIcon component', () => {
  const handleDelete = jest.fn()

  it('renders without crashing', function () {
    render(<DeleteIcon handleDelete={handleDelete} id="1234" />)
  })

  it('matches snapshot', () => {
    const { asFragment } = render(
      <DeleteIcon handleDelete={handleDelete} id="1234" />,
      {}
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
