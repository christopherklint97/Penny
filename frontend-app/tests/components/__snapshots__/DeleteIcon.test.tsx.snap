// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteIcon component matches snapshot 1`] = `
<DocumentFragment>
  <svg
    class="fill-current text-red-700 text-right inline-block mb-1 text-opacity-70 cursor-pointer"
    fill="black"
    height="25px"
    id="1234"
    viewBox="0 0 24 24"
    width="25px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="pointer-events-none"
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      class="pointer-events-none"
      d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
    />
  </svg>
</DocumentFragment>
`;
